src/julia-syntax.scm

```diff
@@ -1714,12 +1714,14 @@
      ((eq? expr argname)
```

Comment:
Ah, looks like another case where we will need to eventually deprecate `_`.

Reply:
I added a test for this one, since I was not sure it needed to be deprecated. But we can decide that later.